broadinstitute/pearl
# [JN-132, JN-374] Create dataset with a real schema + support dataset descriptions

TO TEST:

1) Launch ApiAdminApi, populated with OurHealth
2) Launch admin UI
3) Create new dataset (could take up to 5 mins to show up-> improvements to this piece are in another pending PR)
4) Click to view the dataset in TDR, and verify that all of the columns are there
